### DISCUSSION TITLE
Face Opacity

### DISCUSSION THREAD

**Discussion by hacksider** [hacksider/Deep-Live-Cam#682] [Bounty]
**Feature:** a list of dropdown (incremental by 10, example 10, 20, 30... 100) or a slider to choose the opacity of the webcam from original face to the rendered one to seemingly blend the face of the user and the target image.

**Bounty:** $50 paid via paypal

**Comment by hacksider** [comment]
Now this is fast
https://github.com/hacksider/Deep-Live-Cam/pull/685

**Reply by slmslm330** [reply]
Hello, is it normal that with the original web cam, the image is not sharp with pixels and a generation visible to the eye? I have a TUF 5 Nitro VRAM 6Go

**Comment by hacksider** [comment]
Rewarded to @KRSHH

**Reply by AbandonedCart** [reply]
Congrats! That’s a really good feature idea, too.
